Fastapi.code-workspace

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
does this file applicable to everyone or it's only used for your local setup? we should remove it if this meets the latter option

Reply:
Local, I will remove it